packages/tracking_repository/lib/src/landing_page_events.dart

```diff
@@ -0,0 +1,7 @@
+import 'package:tracking_repository/src/track_event.dart';
+
+/// Default events available on the Landing Page
+class LandingPageEvent extends TrackEvent {
```

Comment:
The concept of events are only meant to be used with blocs not cubits.

Reply:
This is an analytics/tracking event not one that refers to Bloc.